index.js

```diff
@@ -500,6 +501,8 @@ HTTP_LIGHTBULB.prototype = {
 
                         if (this.colorTemperature.unit === TemperatureUnit.KELVIN)
                             minValue = Math.floor(1000000 / minValue);
+                        if (this.colorTemperature.unit === TemperatureUnit.PERCENT)
+                            minValue = 0;
```

Comment:
This actually overwrites the input given from the config.
"maxValue" and "minValue" is used to define a custom max or min value for the hap characteristic, in the case, where the lamp does not support the range which hap defines as default (or supports a much bigger range).

"minValue" (or "maxValue") must be set in the unit mired (as used by hap), so setting it to 0 (or 100) is simply wrong. Also when defining a minValue or maxValue it does not make sense to define that in percent (so the README is probably also misleading). Probably best would be to define the minValue and maxValue in mired(?) [which would basically define what 0% and/or 100% refer to] and document it that way.

Reply:
Apologies, didn't realize that minValue and maxValue must be set in mired.